jenkins/scripts/integration_test.sh

```diff
@@ -88,10 +89,11 @@ TEST_EXECUTER_IP="$(openstack port show -f json "${TEST_EXECUTER_PORT_NAME}" \
 
 if [[ "$OS_REGION_NAME" != "Kna1" ]]
 then
-  # Fetch Free floating IP
-  # TODO: To avoid jobs taking the same floating IP, instead jobs should create / delete / cleanup
-  # their own floating IPs.
-  FLOATING_IP="$(openstack floating ip list --status DOWN -c "Floating IP Address" -f value | shuf -n 1 )"
+  # Create floating IP
+  FLOATING_IP="$(openstack floating ip create -f json \
+    --tag "${TEST_EXECUTER_FIP_TAG}" \
+    --subnet "$(get_subnet_name "${CI_EXT_NET}")" \
+    "${CI_EXT_NET}" | jq -r '.name')"
```

Comment:
`-f value -c name` would get the name of the floating IP without shelling out to `jq`.